src/test/java/io/kaoto/camelcatalog/generators/CamelCatalogSchemaEnhancerTest.java

```diff
@@ -339,10 +339,52 @@ void shouldSetExpressionFormatInExpressionAnyOfFromSaga() {
                 .withObject("definitions")
                 .withObject("org.apache.camel.model.PropertyExpressionDefinition");
 
-        camelCatalogSchemaEnhancer.fillExpressionFormatInOneOf(propertyExpressionNode);
+        camelCatalogSchemaEnhancer.fillModelFormatInOneOf(propertyExpressionNode);
 
         var firstOneOfNode = propertyExpressionNode.withArray("anyOf").get(0);
 
         assertTrue(firstOneOfNode.has("format"));
```

Comment:
These `assertTrue` checks we have the `format` attribute, would it be possible to also check the value of it?